great_expectations/core/expectation_suite.py

```diff
@@ -94,7 +95,7 @@ def __init__(
         self,
         expectation_suite_name: str,
         data_context: Optional[AbstractDataContext] = None,
-        expectations: Optional[List[Union[dict, ExpectationConfiguration]]] = None,
+        expectations: Optional[Sequence[Union[dict, ExpectationConfiguration]]] = None,
```

Comment:
Curious what led to this change...

Reply:
`mypy` error - here's a Slack thread that describes it in more detail: https://greatexpectationslabs.slack.com/archives/C026YSF2JA3/p1675095953275879